package.json

```diff
@@ -24,12 +24,15 @@
         "@easyops-cn/docusaurus-search-local": "^0.44.5",
         "@mdx-js/react": "^3.0.0",
         "@svgr/webpack": "^6.2.1",
+        "@tippyjs/react": "^4.2.6",
```

Comment:
Is tippy necessary? It feels like docusaurus would have something for this

Reply:
I looked at the other articles on the site and couldn't find anything like it